include/dp_log.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 #define _DP_LOG_INT(KEY, VALUE) KEY, _DP_LOG_FMT_INT, VALUE
 #define _DP_LOG_UINT(KEY, VALUE) KEY, _DP_LOG_FMT_UINT, VALUE
 #define _DP_LOG_IPV4(KEY, VALUE) KEY, _DP_LOG_FMT_IPV4, VALUE
-#define _DP_LOG_IPV6(KEY, VALUE) KEY, _DP_LOG_FMT_IPV6, VALUE
```

Comment:
This is done to force the compiler to check the type. Previously it was possible to pass any pointer to `DP_LOG_IPV6()` and the logger would then read 16 bytes from the stack, without care.

Now the compiler will stop you from passing anything else than a union/structure with a member `bytes`, which is still not perfect, but the best I could do in this crazy macro setup for structured logging.